rust-runtime/aws-smithy-http/src/callback/checksum.rs

```diff
@@ -0,0 +1,260 @@
+/*
```

Comment:
we should probably keep these in another crate we can pull in when checksums are actually used